pypeerassets/__main__.py

```diff
@@ -108,7 +108,7 @@ def _increase_fee_and_sign(provider: Provider, key: Kutil, change_sum: Decimal,
     return signed
 
 
-def find_deck(provider: Provider, key: str, version: int, prod=True) -> list:
+def find_deck(provider: Provider, key: str, version: int, prod=True) -> Deck:
```

Comment:
This actually returns list, check it. It has to be like that as sometimes user will search by non-unique key.

Reply:
Oh, well, the reason I changed it was a test was failing because it was trying to index a Deck object... so I'm guessing it used to return a list but as it's written now it's sending back a single Deck:

find_deck -> deck_parser -> Deck (or None on exception)

https://github.com/PeerAssets/pypeerassets/blob/master/pypeerassets/__main__.py#L121
https://github.com/PeerAssets/pypeerassets/blob/master/pypeerassets/__main__.py#L30